py/server/tests/test_parquet.py

```diff
@@ -142,6 +146,70 @@ def test_big_decimal(self):
         self.assertTrue(os.path.exists(file_location))
         shutil.rmtree(base_dir)
 
+    def test_round_trip_data(self):
+        """
+        Pass data between DH and pandas via pyarrow, making sure each side can read data the other side writes
+        """
+
+        # create a table with columns to test different types and edge cases
+        dh_table = empty_table(20).update(formulas=[
+            "someStringColumn = i % 10 == 0?null:(`` + (i % 101))",
+            "nonNullString = `` + (i % 60)",
+            "nonNullPolyString = `` + (i % 600)",
+            "someIntColumn = i",
+            "someLongColumn = ii",
+            "someDoubleColumn = i*1.1",
+            "someFloatColumn = (float)(i*1.1)",
+            "someBoolColumn = i % 3 == 0?true:i%3 == 1?false:null",
+            "someShortColumn = (short)i",
+            "someByteColumn = (byte)i",
+            "someCharColumn = (char)i",
+            # TODO pyarrow indicates this value is out of the allowed range
```

Comment:
going to create one issue for this and the BigDecimal tickets below, since it might be that we just can't round trip some of these types through parquet, python or no python.